src/celery/client.ts

```diff
@@ -109,5 +109,9 @@ export default class Client extends Base {
         taskId = taskId || v4()
         const message = this.createTaskMessage(taskId, taskName, args, kwargs)
         this.sendTaskMessage(taskName, message)
+            .then(() => true)
+            .catch((error) => {
+                throw error
+            })
```

Comment:
This is a bit ugly, but I guess making `sendTask` async would require a lot of `await`s "up" the line?

Reply:
This should still run in the background, otherwise with redis being as flaky as it is, everything will come to a halt here if we await for it.

I found an easier way to ignore the returned promises though: `void`. Check the latest commits.